src/test/regress/sql/resync_metadata.sql

```diff
@@ -0,0 +1,132 @@
+CREATE SCHEMA resync_metadata;
```

Comment:
Can we update the name by including 'sequence' in it? Beside, can we add a comment to test to show why we have it?